src/main/resources/avro/bdg.avdl

```diff
@@ -748,299 +748,304 @@ record Variant {
+   Per-sample reverse strand reference read depth. VCF genotype reserved key ADR, Number=R,
+   split for multi-allelic sites into single integer values for the reference allele
+   (referenceReverseReadDepth, this field) and the alternate allele (alternateReverseReadDepth).
    */
-  union { null, boolean } filtersPassed = null;
+  union { null, int } referenceReverseReadDepth = null;
 
+  // DP Number=1 Type=Integer
   /**
-   Zero or more filters that failed for this genotype call from FORMAT field "FT".
+   Read depth at this position for this sample. VCF genotype field reserved key DP.
    */
-  array<string> filtersFailed = [];
+  union { null, int } readDepth = null;
 
+  // EC Number=A Type=Integer
+  // Comma separated list of expected alternate allele counts for each alternate allele in the same
+  // order as listed in the ALT field. Typically used in association analyses.
   /**
-   True if the reads covering this site were randomly downsampled to reduce coverage.
+   The expected dosage of the alternate allele in this sample. VCF genotype reserved key EC,
+   Number=A, split for multi-allelic sites into a single float value.
    */
-  union { null, boolean } downsampled = null;
+  union { null, float } expectedAlleleDosage = null; // or expectedAlleleCount? see Variant.alleleCount
 
+  // GL Number=G Type=Float
   /**
-   The Wilcoxon rank-sum test statistic of the base quality scores. The base quality
-   scores are separated by whether or not the base supports the reference or the
-   alternate allele.
+   Genotype likelihoods comprised of floating point log10-scaled likelihoods for all possible
+   genotypes given [the set of alleles defined in the REF and ALT fields]. VCF genotype field
+   reserved key GL, Number=G.
    */
-  union { null, float } baseQRankSum = null;
+  array<float> likelihoods = [];
 
-  /**
-   The Fisher's exact test score for the strand bias of the reference and alternate
-   alleles. Stored as a phred scaled probability. Thus, if:
-
-   * a = The number of positive strand reads covering the reference allele
-   * b = The number of positive strand reads covering the alternate allele
-   * c = The number of negative strand reads covering the reference allele
-   * d = The number of negative strand reads covering the alternate allele
-
-   This value takes the score:
-   
-   -10 log((a + b)! * (c + d)! * (a + c)! * (b + d)! / (a! b! c! d! n!)
-
-   Where n = a + b + c + d.
-   */
-  union { null, float } fisherStrandBiasPValue = null;
+  // GLE Number=1 Type=String
+  // Genotype likelihoods of heterogeneous ploidy, used in presence of uncertain copy number.
+  // Reserved genotype field key in VCF v4.2, dropped in VCF v4.3
 
+  // GP Number=G Type=Float
   /**
-   The root mean square of the mapping qualities of reads covering this site.
+   Genotype posterior probabilities in the range 0.0 to 1.0 using the same ordering as the
```

Comment:
Let me know if I have this right:

Field | Cardinality | Action 
----- | ---------- | ------  
`likelihoods` | GL Number=G Type=Float | Convert GL from log10 scale to log scale, optionally convert PL from phred scale to log scale
`nonReferenceLikelihoods` | Number="the ploidy at this site, plus 1" Type=Float | Log scaled, create new VCF key
`genotypePriors` | Number=G Type=Float | Log scaled, rename `priors`, create new VCF key
`genotypePosteriors` | Number=G Type=Float | Log scaled, rename `posteriors`, convert GP from log10 scale to log scale
`posteriorProbabilities` | GP Number=G Type=Float | Drop in favor of log scale `posteriors`